spec/controllers/proposals_controller_spec.rb

```diff
@@ -91,16 +91,22 @@
       end
     end
 
-    context 'details beta' do
+  end
+
+  describe '#show + new details' do
+    before do
```

Comment:
I know we have a lot of `before` blocks in this spec file... I prefer to avoid `before` blocks because I feel that they end up adding bloat to specs (you create data you don't need for subsequent specs) and make them harder to read... so if it were me, I would include these steps in the test itself rather than in a `before`